src/core/hyper-ref/types.js

```diff
@@ -32,10 +32,12 @@ export type State = {|
 export const ATTRIBUTES = {
   ACTION: 'action',
   DELAY: 'delay',
+  EVENT_NAME: 'event-name',
```

Comment:
Is this intended? I couldn't figure it out why this attribute is added now.

Reply:
yeah, I added these attributes as they are valid behavior attributes, and this list is used when generating an `id` for `VisibilityDetectingView` - I figured I'd add these to increase chances of generating a unique id